genkit-tools/common/src/types/eval.ts

```diff
@@ -21,6 +21,33 @@ import { ListEvalKeysRequest, ListEvalKeysResponse } from './apis';
  * This file defines schema and types that are used by the Eval store.
  */
 
+/**
+ * Structured input for eval:flow
+ */
+export const EvalFlowStructuredInputSchema = z.object({
+  cases: z.array(
```

Comment:
Cases is pretty generic. Are we happy with this? Or should we think of something more specific?

Reply:
Renamed to `samples`